src/gax-internal/src/grpc.rs

```diff
@@ -232,14 +220,18 @@ impl Client {
         Ok(headers)
     }
 
-    fn get_retry_policy(
-        &self,
-        options: &gax::options::RequestOptions,
-    ) -> Option<Arc<dyn RetryPolicy>> {
+    fn get_retry_policy(&self, options: &gax::options::RequestOptions) -> Arc<dyn RetryPolicy> {
         options
             .retry_policy()
             .clone()
             .or_else(|| self.retry_policy.clone())
+            .unwrap_or_else(|| {
+                Arc::new(
+                    Aip194Strict
+                        .with_attempt_limit(10)
+                        .with_time_limit(Duration::from_secs(5 * 60)),
+                )
+            })
```

Comment:
This suggests `self.retry_policy` should not be Option<> and be initialized to a value.

Reply:
Agreed. All of the policies are like this, so I am going to leave it as-is in this PR.

I will send a follow up PR to default all of the policies in `*Client::new()` instead of in the helper.

Reply:
#2862